fontforgeexe/startui.c

```diff
@@ -393,7 +393,9 @@ exit( 0 );		/* But the event loop should never return */
 }
 
 static void start_splash_screen(void){
+
     GDrawSetVisible(splashw,true);
+
     GDrawSync(NULL);
```

Comment:
Likewise, adding whitespace to unrelated areas of the code just makes `git blame` harder to read.

Reply:
Oh gosh thought I got them all, yep, will figure out mah editor =PP